testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -118,6 +118,21 @@ def test_rms_fit_trj(self):
         self._assert_rmsd(fitted, 0, 6.929083044751061)
         self._assert_rmsd(fitted, -1, 0.0)
 
+    def test_AlignTraj(self):
```

Comment:
Yes I can do that. Sorry the logging was for my own debugging. I'll get rid
of it in a new commit in a second. Out to lunch.
On Wed, May 25, 2016 at 12:41 PM kain88-de <notifications@github.com> wrote:

> In testsuite/MDAnalysisTests/analysis/test_align.py
> <https://github.com/MDAnalysis/mdanalysis/pull/856#discussion_r64639587>:
>
> > @@ -118,6 +118,21 @@ def test_rms_fit_trj(self):
> >          self._assert_rmsd(fitted, 0, 6.929083044751061)
> >          self._assert_rmsd(fitted, -1, 0.0)
> >
> > +    def test_AlignTraj(self):
>
> can you also test that all warnings/exceptions are raised. The best thing
> is to make sure with coverage that the whole class is covered.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/856/files/bce9e5a5188c94179928a310c3ceb3a02d082b04#r64639587>
>
-- 
Have a wonderful day,
John J. Detlefs